Updated to 1.21.4 in a totally simple way. Additionally I've added fluent support for up to 4 hotbars

UnicornMaster-dev opened the pull request:
It no longer switches the hotbar properly when opening and closing the inventory, try as I might I can't figure it out. Don't know if its a deal breaker but this is what I tried.

unilock commented:
"100% no unnecessary stuff"? There are a lot of code formatting changes in this PR. Reverting them would make it easier to review and merge this.

UnicornMaster-dev commented:
I have really tried searching and I cant find any code formatting changes what are you talking about specifically?

UnicornMaster-dev commented:
New commit adding a changeable number of hotbars up to 4

unilock commented:
> I have really tried searching and I cant find any code formatting changes what are you talking about specifically?

In a few places, line indents were added:
https://github.com/DenWav/ExtendedHotbar/commit/6e4c3d7b41e2cbb3780f2c3d51967177a60537c9#diff-be182288ff1314f3b3bcd45219ebb12dd61a6d51094d72dae39d025d354004e9R111-R114

and annotation values were re-sorted:

https://github.com/DenWav/ExtendedHotbar/commit/6e4c3d7b41e2cbb3780f2c3d51967177a60537c9#diff-1edf1d2efae8de8c16e3b0868917db512f46f44a8ef7e0247480f1f4fa1a34a7R105-R110

...but, honestly, it's not really that much of an issue. Although I do still think reverting these changes would make the PR easier to merge.

As for the contents of the PR... is everything tested and working as intended? Was the hotbar swapping when opening / closing the inventory fixed?

UnicornMaster-dev commented:
I've done a new commit that hopefully fixed the indentation and removed some code I forgot to remove in old versions, Additional I have fixed the hotbar swapping when opening / closing inventory. It works for all 4 hotbars.

UnicornMaster-dev commented:
The workflow build seems to fail cuz I forgot the license header. But I have done a new commit to fix it!
